docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -6,6 +6,140 @@ description: "Review which adjustments must be made to migrate from Camunda 8.7.
+
+:::important
+To reduce the potential of building a backlog,
+the user needs to make sure to drain the backlog queue as much as possible before the update.
+:::
+
+The backlog can be validated via the following Prometheus metrics:
+
+- `operate_import_time_seconds_sum`
+- `operate_import_time_seconds_count`
+
+For example, the following query shows the import latency. If the result is in a range of seconds, it is fine to update.
+
+```promql
+sum(rate(operate_import_time_seconds_sum{namespace=~"$namespace", partition=~"$partition", pod=~"$pod"}[$__rate_interval]))
+/ sum(rate(operate_import_time_seconds_count{namespace=~"$namespace", partition=~"$partition", pod=~"$pod"}[$__rate_interval]))
+```
+
+#### Data migration
+
+To enable Tasklist to utilize the new harmonized indices, Camunda 8.8 provides a migration application to enhance existing process models. This migration application runs asynchronously to the other applications until the importers are done, and ensures that all processes created in 8.7 are migrated and enhanced.
+
+The migration application can be found in the `camunda/bin/migrate` directory.
+
+:::note
+When using the Camunda 8 Helm charts, the migration application will automatically deploy with the importers.
+:::
+
+#### Turn off importers after completion
+
+Importers are only necessary at the start of the migration. Once complete, they can be safely turned off.
+
+To detect whether importers are done, users can validate the state in the Elasticsearch/OpenSearch index:
+
+- `tasklist-import-position-8.2.0_`
+- `operate-import-position-8.3.0_`
+
+If all entries show the `completed` property set to `true`, the importing is done.
+
+Alternatively, metrics can be validated via:
+
+- `operate_import_completed`
+- `tasklist_import_completed`
+
+These properties can be used in the following example query:
+
+```
+sum(tasklist_import_completed{namespace="$namespace"}) by (importPositionAlias)
+```
+
+![metrics-import-complete](./img/import-complete-metrics.png)
+
+#### Update any custom prefixes
+
+:::important
+This step is required if you use an installation with custom prefixes.
+:::
+
+The new harmonized index schema introduces one common index prefix. To update to Camunda 8.8, indices must first migrate to one common index, so the Camunda Exporter, REST API, and other consumers can be configured correctly.
+
+##### Prefix migration
+
+:::warning
+Disk usage will massively increase while the migration is ongoing (and will return to normal when complete).
+Ensure you have at least double the disk space free.
+:::
+
+:::note
+While we test and validate the provided migration script,
+it is recommended that you first back up the database, and test the migration script in a test environment first
+to ensure there are no unexpected issues.
+:::
```

Comment:
Would it be a good practice to always test the script before running it, even after the migration is out of our own test/validate phrase? We could remove the first portion of the sentence if so.

Reply:
Not fully sure, whether I understand what you mean. But I guess we can improve this note yes.

Reply:
Improved https://github.com/camunda/camunda-docs/pull/5023/commits/52bf2add93d31034f16c11d2727d80be8dc108ed